scripts/user_env/0_nodenv.sh

```diff
@@ -12,15 +12,19 @@ git clone --depth 1 https://github.com/nodenv/nodenv-default-packages.git "${NOD
 
 ( cd ${NODENV_ROOT} && src/configure && make -C src )
 
-# install package managers in all nodenv versions
-echo "npm" >> "${NODENV_ROOT}/default-packages"
+# Install yarn package manager in all nodenv versions
+# Leave npm version aligned with version of node
 echo "yarn" >> "${NODENV_ROOT}/default-packages"
```

Comment:
If its a problem for npm might as well remove  it for yarn

Reply:
That will break things like lerna which kind of expect yarn to be there. I think we leave it in until it becomes an issue.